### DISCUSSION TITLE
Pasted text in SQL editor remains selected and is ugly orange color

### DISCUSSION THREAD

**Discussion by vincenzon** [supabase/supabase#20868] [Questions]
When I paste code into the SQL editor, it has this ugly orange color that never seems to go away, after typing in the selection or around it: 

![image](https://github.com/supabase/supabase/assets/26041391/f1460fb9-4b80-4679-9090-ca7351a68a2e)

is there some setting (in supabase, in the browser, in the OS, ...?) that will fix this? It is sometimes hard to read the affected code. 

Thanks for any help or suggestions.

**Comment by GaryAustin1** [comment]
You might create a dashboard issue with specifics on browser/OS and an example in supabase/supabase issues.  I just pasted and don't see this, but I have seen some strange highlighting in the past.
